packages/react-vapor/src/components/select/MultiSelectConnected.tsx

```diff
@@ -110,7 +111,7 @@ class MultiSelect extends React.PureComponent<IMultiSelectProps> {
             <div className="flex flex-row flex-center sortable-selected-option" key={item.value}>
                 <span className="mr1 text-medium-grey">{index + 1}</span>
                 <DraggableSelectedOption
-                    label={item.displayValue || item.value}
+                    label={item.selectedDisplayValue || item.displayValue || item.value}

```

Comment:
It might not be relevant in that case, but it could be interesting to use nullish coalescing operator instead of the "OR" logical operator. It would look like this `item.selectedDisplayValue ?? item.displayValue ?? item.value`
Here's some [info](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) explaining this suggestion.

Reply:
Very interesting, thank you for the suggestion!